### PR TITLE
change-prefix-extention

### PR DESCRIPTION
**Pull request by Ynoctis**
**Patch description**
Added Prefix extension to cogs that allows the user to customize the prefix of the bot

**Testing steps**
image

**Other information**
In future PR, will add a local database so that when the bot reloads it will use the customized prefix

### CODE REVIEW

File: cogs/change_prefix.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    @commands.command()
+    async def change_prefix(self, ctx, new_prefix):
+        """
+        Changes the command prefix of this bot
+        """
+        if len(new_prefix) == 1:
+            self.bot_client.command_prefix = new_prefix
+            await ctx.send(f"The prefix of this bot has now been changed to {new_prefix}")
+        else:
+            await ctx.send("The entered prefix has a length greater than 1. "
```
Comment (mus1cholic) [inline cogs/change_prefix.py:18]: nit: `len(new_prefix)` might be 0 potentially? add edge case for that

**Review by mus1cholic — CHANGES_REQUESTED** [review]
Good change, but as we have discussed before we want to modularize our code into categories. In the wiki, the command `change_prefix` is under the category `misc`, so it would be nice if you can port your command over to that

File: cogs/change_prefix.py (hunk 1 above)
Comment (Ynoctis) [reply cogs/change_prefix.py]: fixed

**Review by mus1cholic — APPROVED** [review]
Looking good

**Pull request merged**